e2e/regional_DR_test.go

```diff
@@ -0,0 +1,26 @@
+package e2e_test
```

Comment:
I would just call this example_test.go for now. We don't want to have all regional dr tests in one module, and most tests should be compatible with all environments, so we don't want to focus on the environment type in the test names.